sql-db/hibernate-reactive/src/main/java/io/quarkus/ts/reactive/GroundedEndpoint.java

```diff
@@ -0,0 +1,94 @@
+package io.quarkus.ts.reactive;
+
+import javax.ws.rs.Consumes;
+import javax.ws.rs.GET;
+import javax.ws.rs.POST;
+import javax.ws.rs.PUT;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+
+import org.hibernate.reactive.mutiny.Mutiny;
+
+import io.quarkus.arc.Arc;
+import io.smallrye.mutiny.Uni;
+
+@Path("/hibernate")
+@Produces(MediaType.APPLICATION_JSON)
+@Consumes(MediaType.APPLICATION_JSON)
+public class GroundedEndpoint {
```

Comment:
Just a suggestion to stick with Resource for class name.
What is the purpose of this class?
Does it lack Multi(Response) methods?

Reply:
It doesn't use Panache methods, hence the pun in the name(ideas for better name are welcomed, though). I am not sure, that Resource can be a good name for an endpoint class.